AdobeStockImageAdminUi/Test/Mftf/ActionGroup/AdminSaveCmsPageAndAssertImagePreviewActionGroup.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminSaveCmsPageAndAssertImagePreviewActionGroup">
+        <click selector="{{AdobeStockSection.imagesThumbnails}}" stepKey="clickOnThumbnail"/>
```

Comment:
Please fix this selector as well. (check one of previous comments)

Reply:
ok